File: fizz_buzz.py
def func_fizz_buzz(n):
    
    fizzBuzz = []
    for i in range(1,n+1):
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")
            fizzBuzz.append("FizzBuzz")
        elif i % 3 == 0:
            print("Fizz")
            fizzBuzz.append("Fizz")
        elif i % 5 == 0: 
            print("Buzz")
            fizzBuzz.append("Buzz")
        else:
            print(i)
            fizzBuzz.append(i)
    return fizzBuzz

if name == __main__:
    func_fizz_buzz(100)  

File: fizz_buzz.php
<?php

function fizzBuzz($n){
    $fizzbuzzArraz = array();
for ($i = 1; $i <= $n; $i++) {
    if($i % 3 == 0 && $i % 5 == 0){
        echo "FizzBuzz";
        echo "<br>";
        $fizzbuzzArraz[$i] = "FizzBuzz";
    }
    elseif($i % 3 == 0){
       echo "Fizz";
       echo "<br>";
       $fizzbuzzArraz[$i] = "Fizz";
    }
    elseif($i % 5 == 0){
        echo "Buzz";
        echo "<br>";
        $fizzbuzzArraz[$i] = "Buzz";
    }
    else{
        echo $i;
        echo "<br>";
        $fizzbuzzArraz[$i] = $i;
    }
    }
    return $fizzbuzzArraz;
}
fizzBuzz(15);
?>
